agents-api/agents_api/models/chat/prepare_chat_context.py

```diff
@@ -37,6 +37,7 @@
         "session": make_session(
             agents=[a["id"] for a in d["agents"]],
             users=[u["id"] for u in d["users"]],
+            updated_at=d["session"].pop("updated_at") / (1000000.0),
```

Comment:
This was my motive:

The `created_at` & `updated_at` fields in all relations are stored in cozo in nanoseconds and as a `float` (example value is: `1734460003.168783`). However, there is one exception. The type of the `updated_at` column in the `sessions` relation is a `Validity` instead of a `float`. And timestamp part of the `updated_at` is getting floored by default (see [here](https://github.com/julep-ai/julep/blob/c61f26c2688755bd8a457b996a646463ddbfab7c/agents-api/agents_api/models/utils.py#L112) and [here](https://github.com/julep-ai/julep/blob/c61f26c2688755bd8a457b996a646463ddbfab7c/agents-api/migrations/migrate_1733493650_add_recall_options_to_sessions.py#L35)), not sure why but my guess is that Validity .

So when a session is created, it'll have the following stored in cozo:

`created_at`: `1734460003.168783`
`updated_at`: `1734460003`

However, when trying to update a session, let's say at timestamp `1734460174.546739`, for some reason this number isn't floored, but instead, only the floating point is taken away, so the `updated_at` will have a value of `1734460174546739`, which triggers the validation error when it's being converted to a pydantic `AwareDatetime` object.

And tbh this was very tedious to figure out & fix. Please let me know if there's a better solution to this.